CMakeLists.txt

```diff
@@ -74,9 +80,9 @@ endif()
 
 find_package(GSL)
 find_package(LAPACK)
-find_package(Casacore)
+find_package(Casacore REQUIRED)
```

Comment:
Note that this won't make a difference because the `FindCasacore.cmake` module included under `cmake` doesn't check this. I included a fix for this in the `non-cuda-builds` branch, so no need to worry about it here.